### DISCUSSION TITLE
Exterior/Math/P%26S/4/

### DISCUSSION THREAD

**Discussion by giscus** [SAVIA7582/Exterior#4] [General]
# Exterior/Math/P%26S/4/

https://savia7582.github.io/Exterior/Math/P%26S/4/

**Comment by JYbestow** [comment]
**勘误：例1(2)的解析误将原题中的P(X=0,Y=1)打成了P(X=1,Y=0)**

**Comment by LH-0124** [comment]
这一页的文件似乎出了点问题，我的windows系统看不到文件，似乎崩溃了

**Reply by Enxulansis** [reply]
我也是

**Comment by CSGISBME** [comment]
泊松分布的符号和书本不一样。

**Comment by yifanjiang00** [comment]
你好，这一章节“方差的线性组合”公式里协方差前面的系数似乎漏乘了

**Reply by Liang0715** [reply]
是的，第一页末尾“方差的线性组合”公式应该再乘上Ci和Cj。
